### PR TITLE
Added Start/End arguments to :tab-move

### LINKED ISSUE #6776: Add start/end arguments to :tab-move
Hello. Thanks for everything!

By the way, I'm very sorry, but if it's possible, could you please make the default behavior of :tab-move with no argument "move to last tab" like :tabmove in vim?

Also, it seems that :tab-move 1 is used to move to the first tab, not :tab-move 0.
I'm guessing that's how it was made to match Chrome's Alt+1 key binding.

How about the idea of brushing up the specification, at least for the parts where the tab behavior is inconsistent, using the following vim help?

- https://vimhelp.org/tabpage.txt.html

I hope you will consider it.

### PR DESCRIPTION
**Pull request by harish3124**
# fix #6776

### CODE REVIEW

File: qutebrowser/browser/commands.py [hunk 1]
```diff
@@ -1004,11 +1004,10 @@ def tab_focus(self, index: Union[str, int] = None,
             raise cmdutils.CommandError("There's no tab with index {}!".format(
                 index))
 
-    @cmdutils.register(instance='command-dispatcher', scope='window')
-    @cmdutils.argument('index', choices=['+', '-'])
-    @cmdutils.argument('count', value=cmdutils.Value.count)
-    def tab_move(self, index: Union[str, int] = None,
-                 count: int = None) -> None:
+    @cmdutils.register(instance="command-dispatcher", scope="window")
+    @cmdutils.argument("index", choices=["+", "-", "start", "end"])
```
Comment (harish3124) [inline qutebrowser/browser/commands.py:1008]: Added Start and End to choices

**Review by The-Compiler — CHANGES_REQUESTED** [review]
Thanks! The implementation looks good - could you please also add a test for this [in tabs.feature](https://github.com/qutebrowser/qutebrowser/blob/master/tests/end2end/features/tabs.feature#L488)?

**Comment by harish3124** [comment]
Sure

**Comment by The-Compiler** [comment]
Looks great now, thank you! Merged with a few minor doc changes in 2a18eeb558d2b4faa33b5dbba62de90d4e27b9e6.

**Pull request merged**